subcommittees/mentoring/roles/mentees.md

```diff
@@ -0,0 +1,17 @@
+# Mentees
+
+## Responsibilities
+
+
+## Time commitment
+
+
+## Requirements
+
+
+
+## To Volunteer
+
```

Comment:
The join the mentoring program, email the Data Carpentry Associate Director at ebecker@datacarpentry.org.